spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/core/SpannerQueryOptions.java

```diff
@@ -40,6 +40,8 @@
 
 	private OptionalLong offset = OptionalLong.empty();
 
+	private boolean allowMissingResultSetColumns;
```

Comment:
please rename to `allowPartialRead` as that's shorter and consistent with the terminology we use in the ref-docs too

Reply:
changes made